venues/aclweb.org/ACL/migrate_submissions_ARR_ACL.py

```diff
@@ -0,0 +1,362 @@
+import argparse
+from re import sub
+import openreview
+from tqdm import tqdm
+import csv
+
+"""
+OPTIONAL SCRIPT ARGUMENTS
+
+    baseurl -  the URL of the OpenReview server to connect to (live site: https://openreview.net)
+    username - the email address of the logging in user
+    password - the user's password
+
+"""
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base URL")
+parser.add_argument('--username')
+parser.add_argument('--password')
+args = parser.parse_args()
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+
+
+# Posting submission invitation 
+submission_invitation = openreview.Invitation(
+    id = "aclweb.org/ACL/2022/Conference/-/Submission",
+    signatures = ["aclweb.org/ACL/2022/Conference"],
+    readers = ["everyone"],
+    invitees=["everyone"],
```

Comment:
```suggestion
    invitees=["aclweb.org/ACL/2022/Conference"],
```

keep the invitation restricted.